interlok-core/src/test/java/com/adaptris/core/MultiPayloadAdaptrisMessageImpTest.java

```diff
@@ -62,5 +67,22 @@ public void testRemovePayloadMessageHeader() {
     assertFalse(multiPayloadAdaptrisMessage.payloadHeadersContainsKey("key"));
     assertFalse(multiPayloadAdaptrisMessage.payloadHeadersContainsKey("another-payload", "key"));
   }
+  
+  //Unit test added to test bug INTERLOK-4325 
+  @Test
+  public void testUpdatingOriginalPayloadFromInputStream() throws IOException {
+	System.out.println(PROPERTIES.getProperty(EXAMPLE_PAYLOAD_FILE));
+	MultiPayloadAdaptrisMessage multiPayloadAdaptrisMessage = MessageHelper.createMultiPayloadMessage("default-payload", PROPERTIES.getProperty(EXAMPLE_PAYLOAD_FILE));
+    System.out.println(multiPayloadAdaptrisMessage.getContent("default-payload"));
```

Comment:
We want to avoid System.out.println.

Reply:
gahhh ailly mistake and forgot ot remove this, removed now!